source/features/comments-time-machine-links.tsx

```diff
@@ -78,7 +99,7 @@ async function showTimemachineBar(): Promise<void | false> {
 		}
 
 		const parsedUrl = new GitHubURL(location.href);
-		parsedUrl.branch = `${parsedUrl.branch}@{${date}}`;
+		parsedUrl.branch = await getDatedSha(parsedUrl, date);
```

Comment:
Can you set the old href first and then update it when the API returns? This way the bar appears faster (and still works most of the time anyway)

Reply:
Yes

Reply:
Done